Interni/NormeProgetto/components/norme-c3.tex

```diff
@@ -0,0 +1,475 @@
+		\G{} & Elenco di tutti i termini presenti nella documentazione che, secondo i membri, necessitano una definizione al fine di chiarirne il significato e rimuovere eventuali ambivalenze. & Esterno \\  			
+		\bottomrule
+		\hiderowcolors
+		\caption{Nome, Descrizione ed uso dei documenti formali prodotti}
+	\end{longtable}
+}
+
+\item \textbf{Informali}: sono documenti non soggetti a versionamento o che non sono ancora stati approvati dal \textit{Responsabile di Progetto}. \newline 
+In particolare, i documenti informali prodotti saranno:
+{
+	\setlength{\freewidth}{\dimexpr\textwidth-1\tabcolsep}
+	\renewcommand{\arraystretch}{1.5}
+	\setlength{\aboverulesep}{0pt}
+	\setlength{\belowrulesep}{0pt}
+	\rowcolors{2}{AzzurroGruppo!10}{white}
+	\begin{longtable}{L{.210\freewidth} L{.6\freewidth} L{.080\freewidth}}
+		\toprule 
+		\rowcolor{AzzurroGruppo!30}
+		\textbf{Nome} & \textbf{Descrizione} & \textbf{Uso}\\
+		\toprule
+		\endhead		
+		Verbali interni & Contiene le informazioni e le decisioni prese durante gli incontri tra i membri del gruppo. & Interno \\ 
+		Verbali esterni & Comprende le informazioni ed i chiarimenti ricevuti durante gli incontri tra i membri ed il committente. & Esterno \\
+		\bottomrule
+		\hiderowcolors
+		\caption{Nome, Descrizione ed uso dei documenti informali prodotti}
+	\end{longtable}
+}
+\end{itemize}
+\subsubsection{Directory di un documento}
+Le directory prendono il nome dal documento contenuto, nella forma \textbf{NomeDelDocumento}.  Questa viene poi collocata,  a seconda del tipo di contenuto,  in una delle directory \textbf{Interni} o \textbf{Esterni}. 
+\subsubsection{Struttura dei documenti}
+\paragraph{Attribuzione del nome}
+\begin{itemize}
+	\item La denominazione dei documenti formali sarà la seguente \newline
+	\textbf{[NomeDocumento]\_v[X].[Y].[Z]-[a].[b]}
+	\begin{itemize}
+		\item \textbf{NomeDocumento} è il nome ufficiale del documento, senza spazi tra le parole e con solamente le iniziali di ogni parola in maiuscolo (convenzione PascalCase);
+		\item \textbf{v[X].[Y].[Z]} con \textbf{v} che abbrevia la parola versione, \textbf{[X]} indica il numero incrementale di revisione del responsabile di progetto, \textbf{[Y]} indica il numero incrementale della verifica, \textbf{[Z]} indica il numero di modifiche apportate al documento, che riparte da 0 dopo ogni verifica ed approvazione.
+		\item \textbf{[a].[b]} con \textbf{A} che indica una versione completa e funzionante del prodotto: le componenti software
+implementano tutti i requisiti obbligatori, i test sono stati tutti superati, le metriche
+di qualità soddisfatte, e sono presenti versioni approvate di tutti i documenti richiesti
+per il prodotto finale; viene incrementato quando il prodotto software non è più
+retrocompatibile; la numerazione parte da 0 e non viene annullata; \textbf{B} cresce al raggiungimento degli obiettivi degli incrementi pianificati nel \PdP{}: poiché tali incrementi presentano obiettivi sia documentali che
```

Comment:
cambiare ; in .

mettere la maiuscola su "La"

Reply:
Outdated, da ricontrollare